### PR TITLE
 0.67.1

### PR DESCRIPTION
**Pull request by persuader72**
Updated rocko to version HA 0.67.1

### CODE REVIEW

**Comment by bachp** [comment]
@persuader72 Thanks for the PR. Two things.

1. The commit message for python-telegram-bot seems to have a leading `-` that is not necessary.
2. Can you do the PR against master too. Master should always be at least the same version as the stable branches.

File: recipes-devtools/python/python-pydeconz.inc [hunk 1]
```diff
@@ -0,0 +1,9 @@
+SUMMARY = "A Python library for communicating with deCONZ REST-API from Dresden Elektronik"
+HOMEPAGE = "https://github.com/Kane610/deconz"
+LICENSE = "MIT"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=4b0970808e9ec547316870e9ed531a2d"
```
Comment (bachp) [inline recipes-devtools/python/python-pydeconz.inc]: This file doesn't seem to exist. Maybe switch to version 0.1.0.dev0

Comment (persuader72) [reply recipes-devtools/python/python-pydeconz.inc]: Sorry, I can't understand.
What is the file taht don't exists?

Comment (bachp) [reply recipes-devtools/python/python-pydeconz.inc]: The tarball that is downloaded from PyPi doesn't seem to contain a `LICENSE` file. Bitbake gives me an error about that.

**Comment by bachp** [comment]
@persuader72 Are you still working on this?

**Comment by bachp** [comment]
There is already a newer version on master. Closing

**Pull request closed without merging**